packages/docs/docs/api/tutorials/bots/logistics-into-ehr.md

```diff
@@ -0,0 +1,153 @@
+
+# Integrating Logistics into your EHR
+
+Adding a logistics provider which ships packaged goods is a common need for medical apps and services.  Common shipments include:
+
+* At home lab tests
+* Durable Medical Equipment (DME) sent to home
+* At-home pharmacy
+
+Many compainies use third party logistics (3PL) to do fulfillment.  Examples of these providers are [Fulfulled By Amazon (FBA)](https://sell.amazon.com/fulfillment-by-amazon) or [ShipBob](https://product.shipbob.com/) that warehouses inventory and sends goods to customers.
+
+## This guide will show you
+
+* How to integrate with a sample 3PL provider (in this case [Easypost](https://www.easypost.com/))
+* How to enable communication between the Easypost (3PL) and Medplum
+* How to link your shipments to your medical records stored in FHIR
+* Security and compliance best practices for the integration.
+
+## The Workflow
+
+When we this implementation is complete, the whole workflow below will be done in an automated way with without human intervention.
+
+* Create a [ServiceRequest](https://app.medplum.com/ServiceRequest) when a package is shipped to a [Patient](https://app.medplum.com/Patient)'s home
+* Link a Tracker url (that shows the delivery history of a package) to a ServiceRequest.
+* Receive updates as that shipment is sent to the Patient via [Easypost Webhooks](https://www.easypost.com/docs/api#webhooks) and update the ServiceRequest accordingly.
+* Receive confirmation that the shipment has arrived (also via Easypost Webhooks).
+
+## The Implementation
+
+### Account and Policy Setup
+
+* Make sure you have an account on Medplum, if not, [register](https://app.medplum.com/register).
+* Make sure you have an account on Easypost, if not, [register](https://www.easypost.com/signup)
+* Create a [ClientApplication](https://app.medplum.com/admin/project) on Medplum called "Easypost Webhook".
+* (Optional) Create a very restrictive [AccessPolicy](https://app.medplum.com/AccessPolicy) called "Easypost", make it so that the policy only allows readwrite on the ServiceRequest.
+* (Optional) In the [ProjectAdmin dashboard](https://app.medplum.com/admin/project) apply the "Easypost" policy to the `ClientApplication` by clicking `Access`.
+
+### Bot Setup
+
+* Linking the logistics system, Easypost, to Medplum is done through [Medplum Bots](https://app.medplum.com/Bot).
+* At a high level, webhooks are sent from Easypost to Medplum, updating data in Medplum based on the contents of the Webhook.
+* This example assumes that there is a `ServiceRequest` with an identifier that matches the Easypost `reference`.
+
+* Make the Easypost subscription bot that will listen for webhooks
+  * First, [create a bot](https://app.medplum.com/Bot/new) called EasyPost Shipment Handler and save it
+  * Paste the code below into the Bot you created and save.
+
+  ```js
+  //Grab the reference and the tracker object out of the Easypost Shipment json
+  const reference = input.get('reference');
```

Comment:
It would be helpful to have a description of the kind of data that comes out of EasyPost, and what each of these fields is